doc/api/fs.md

```diff
@@ -3187,6 +3187,18 @@ first argument. In this case, `fs.ftruncate()` is called.
 Passing a file descriptor is deprecated and may result in an error being thrown
 in the future.
 
+```js
+console.log(fs.readFileSync('/path/to/file', 'utf8'));
+// Prints: Node + Learn
```

Comment:
??????

Reply:
I think that’s supposed to be the contents of the test file.

@johinsDev It might be better to use something more generic here (e.g. `Hello, world!`), and maybe use an `fs.writeFileSync()` call before to make sure that those are actually the file’s contents?